.github/CODEOWNERS

```
@@ -188,6 +186,40 @@
 
 /sound/ @Sadboysuss
 
+# Timothymtorres
+
+/code/modules/basketball/ @timothymtorres
+/_maps/minigame/basketball/ @timothymtorres
+/code/controllers/subsystem/weather.dm @timothymtorres
+/code/modules/admin/verbs/adminweather.dm @timothymtorres
+/code/datums/weather/ @timothymtorres
+/code/datums/components/anti_magic.dm @timothymtorres
+/code/datums/components/rotation.dm @timothymtorres
+/code/datums/ai_laws/ @timothymtorres
+/code/game/objects/items/AI_modules/ @timothymtorres
+/code/modules/mob/living/silicon/laws.dm @timothymtorres
+/code/game/sound.dm @timothymtorres
+/code/game/objects/structures/shower.dm @timothymtorres
+/code/modules/atmospherics/machinery/air_alarm/ @timothymtorres
+/code/datums/skills/athletics.dm @timothymtorres
+/code/modules/mob/living/death.dm @timothymtorres
+/code/modules/cards/ @timothymtorres
+/code/game/machinery/barsigns.dm @timothymtorres
+/code/game/machinery/slotmachine.dm @timothymtorres
+/code/game/objects/items/devices/flashlight.dm @timothymtorres
+/code/game/say.dm @timothymtorres
+/code/modules/mob/living/carbon/carbon_say.dm @timothymtorres
+/code/modules/language/ @timothymtorres
+/code/datums/status_effects/debuffs/tower_of_babel.dm @timothymtorres
+/code/modules/events/wizard/tower_of_babel.dm @timothymtorres
+/code/modules/spells/spell_types/tower_of_babel.dm @timothymtorres
+/code/game/objects/effects/spawners/ @timothymtorres
+/code/datums/diseases/ @timothymtorres
+/code/datums/mood_events/ @timothymtorres
+/code/game/objects/items/cigs_lighters.dm @timothymtorres
+/code/game/objects/structures/statues.dm @timothymtorres
+/code/datums/elements/can_barricade.dm @timothymtorres
```

Codeowners exists purely to request reviews from certain developers, not to display who wrote what, we have blame for that - a lot of these don't make much sense. You'd just be flooded with requests for minor changes.

```suggestion
/code/modules/basketball/ @timothymtorres
/_maps/minigame/basketball/ @timothymtorres
/code/controllers/subsystem/weather.dm @timothymtorres
/code/modules/admin/verbs/adminweather.dm @timothymtorres
/code/datums/weather/ @timothymtorres
/code/datums/mood_events/ @timothymtorres
```

---

By your reasoning none of these make sense either.

---

And FYI I did already curtail this list before I opened this PR. If I included every file I had created or touched, this list would be hundreds of lines long. I mainly included things I had spent MONTHS bug fixing, refactoring, etc.